locustempus/templates/django_registration/activation_failed.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% block title %}&mdash; {% if account %}Activation complete{% else %}Activation problem{% endif %}{% endblock %}
+
+{% block registration_content %}
+<h2 class="pagetitle signuptitle">Account Activation</h2>
+
+<div><p>Oops &ndash; it seems that your activation key is invalid.  Please check the url again.</p></div>
```

Comment:
remove "it seems that"...and I think there are two spaces before Please rather than 1.